source/simulator/helper_functions.cc

```diff
@@ -23,7 +23,16 @@
 #include <aspect/melt.h>
 #include <aspect/global.h>
 
+#include <aspect/geometry_model/interface.h>
+#include <aspect/heating_model/interface.h>
 #include <aspect/heating_model/adiabatic_heating.h>
+#include <aspect/material_model/interface.h>
+#include <aspect/particle/generator/interface.h>
+#include <aspect/particle/integrator/interface.h>
+#include <aspect/particle/interpolator/interface.h>
+#include <aspect/particle/output/interface.h>
+#include <aspect/prescribed_stokes_solution/interface.h>
+#include <aspect/postprocess/visualization.h>
```

Comment:
In `simulator_access.cc` we assume that all of these are included in `simulator.h` and therefore do not list them explicitly. Is there a reason you want to handle this differently here?